docs/jwst.rst

```diff
@@ -54,14 +54,31 @@ All classes share some common attributes:
     available modes well enough to be aware if they are trying to
     simulate an inconsistent or physically impossible configuration. 
 
+Optical Telescope Element (OTE)
+===============================
+
+The JWST Optical Telescope Element consists of the telescope optics that serve all the science instruments and the fine guidance sensor. Most notably, this means the primary, secondary, and tertiary mirrors. The OTE contributes to the overall wavefront error (and therefore the aberrations in instrument PSFs) in a few ways:
+
+  * The limits of precisely manufacturing the mirrors introduce tiny high spatial frequency bumps and ripples of optical path difference
+  * During commissioning, the telescope mirror segments will be aligned and phased as precisely as possible, but small errors in the final aligned configuration will still contribute to WFE
+  * The WFE will vary with field position, which is inherent in the OTE optical design even if perfectly aligned
+  * Aberrations can be introduced by pupil shear or other misalignments between the OTE and each science instrument
+
+Let none of this make you think the JWST optics are deficient in some way! JWST is an extremely precisely engineered optical system. We nevertheless must model these small effects in order to provide users with the most realistic PSF we can.
+
+These effects are simulated at high fidelity in models maintained by Ball Aerospace, which in turn were used to create the OPD map files for the JWST instruments included in WebbPSF.
+
```

Comment:
Add: 
"Specifically, WebbPSF uses information derived from the as-built OTE optical model Revision G (for the static surface figures of each segments) and the overall JWST optical error budget Revision W (for OTE to ISIM misalignments, WFSC residuals, stability, and budgeted uncertainties for both the OTE and SI contributions.) "